eru/src/views/User/ProfileView.vue

```diff
@@ -5,31 +5,33 @@
       <!-- user profile banner -->
       <!-- if theUser object contain data -->
       <UserBannerProfile
-        v-if="theUser"
-        :key="theUser.user._id"
-        :userId="theUser.user._id"
-        :name="theUser.user.name"
-        :userName="theUser.user.username"
-        :profileImage="theUser.user.avatar"
-        :joinedDate="theUser.user.created_at"
-        :alreadyFollow="theUser.user.is_followed_by_current_screeler"
-        :alreadyFollowedByThatUser="theUser.user.is_following_current_screeler"
-        :followingsCount="theUser.user.followings_count"
-        :followersCount="theUser.user.followers_count"
+        v-if="userInView"
+        :key="userInView.user._id"
+        :userId="userInView.user._id"
+        :name="userInView.user.name"
+        :userName="userInView.user.username"
+        :profileImage="userInView.user.avatar"
+        :joinedDate="userInView.user.created_at"
+        :alreadyFollow="userInView.user.is_followed_by_current_screeler"
+        :alreadyFollowedByThatUser="
+          userInView.user.is_following_current_screeler
+        "
+        :followingsCount="userInView.user.followings_count"
+        :followersCount="userInView.user.followers_count"
       />
-      <!--/ if theUser object contain data -->
+      <!--/ if userInView object contain data -->
```

Comment:
I used a name which was more descriptive. "theUser" is kind of vague.

Reply:
I created a userInView object in the store, this object keeps the current user whose profile is being viewed. I did this so that when you click on the follow button the number of followers on the ProfileViewCard should be updated

Reply:
> I used a name which was more descriptive. "theUser" is kind of vague.

Maybe but Next time, please don't change existing variable name like that, so for this time it's ok

Reply:
> > I used a name which was more descriptive. "theUser" is kind of vague.
> 
> Maybe but Next time, please don't change existing variable name like that, so for this time it's ok

ok noted